roles/signals.py

```diff
@@ -0,0 +1,18 @@
+"""
+Signals handlers
+"""
+from __future__ import unicode_literals
+
+from django.db.models.signals import post_migrate
+from django.dispatch import receiver
+
+from roles.utils import sync_groups_permissions
+
+
+# pylint: disable=unused-argument
+@receiver(post_migrate)
+def my_callback(sender, **kwargs):
```

Comment:
I guess it is technically possible, but I need more time to investigare and I think I spent enough on this. If you have an example working with migrations I can try.